examples/Console/TestMetrics.cs

```diff
@@ -32,8 +33,38 @@ internal static object Run(MetricsOptions options)
                 .AddSource("TestMeter") // All instruments from this meter are enabled.
                 .SetDefaultCollectionPeriod(options.DefaultCollectionPeriodMilliseconds)
                 .AddProcessor(new TagEnrichmentProcessor("resource", "here"))
+
+                // Add multiple exporters
                 .AddExportProcessor(new MetricConsoleExporter("A"))
                 .AddExportProcessor(new MetricConsoleExporter("B"), 5 * options.DefaultCollectionPeriodMilliseconds)
+
+                // Add multiple Views
+
+                .AddView(
+                    (inst) => inst.Name == "counter",
+                    () =>
+                    {
+                        return new OpenTelemetry.Metrics.IAggregator[]
+                        {
+                            new SumMetricAggregator(false, true),
```

Comment:
discussed offline. We want to limit this to simply selecting a well-known built-in aggregator only. We can improve the Views to support more complex scenarios like picking an instance of the aggregator later, as spec evolves.

Reply:
I refactor the public AddView interface to allow a "simple" version. Let me know if this version is more acceptable for now.